app/views/settings_dialog.py

```diff
@@ -686,6 +688,50 @@ def _do_todds_tab(self) -> None:
         )
         group_layout.addWidget(self.todds_overwrite_checkbox)
 
+    def _do_themes_tab(self) -> None:
+        tab = QWidget()
+        self.tab_widget.addTab(tab, "Themes")
+
+        tab_layout = QVBoxLayout(tab)
+        tab_layout.setAlignment(Qt.AlignmentFlag.AlignTop)
+
+        group_box = QGroupBox()
+        tab_layout.addWidget(group_box)
+
+        group_layout = QVBoxLayout()
+        group_box.setLayout(group_layout)
+
+        self.themes_label = QLabel("Themes Support")
+        self.themes_label.setFont(GUIInfo().emphasis_font)
+        group_layout.addWidget(self.themes_label)
+
+        self.enable_themes_checkbox = QCheckBox(
+            "Enable to use .qss Themes (stylesheet) / Disable to use OS/System Theme"
+        )
```

Comment:
Compress the info for the label. Add a tooltip for additional context and info.

Reply:
E.g.,
Label: Use stylesheet instead of system theme

Tooltip: Utilize the selected .qss stylesheet for the theme instead of the system theme

Reply:
Done